MicroLMS/heci/PTHICommand.c

```diff
@@ -202,10 +206,10 @@ AMT_STATUS _call(const unsigned char *command, UINT32 command_size, UINT8 **read
 	{
 		return status;
 	}
-	if ((expSize != 0) && (expSize != outBuffSize))
-	{
-		return PTSDK_STATUS_INTERNAL_ERROR;
-	}
+	//if ((expSize != 0) && (expSize != outBuffSize))
```

Comment:
if it ain't used,  remove it -- history is there if ya need it again.

Reply:
Yep, planned for likely removal. Ensuring behavior stays intact. Took note. Will comment or remove.